build.gradle.kts

```diff
@@ -64,6 +65,7 @@ intellij {
     // javascript
     "JavaScript",
     "Pythonid:$pluginsVersion",
+    "AngularJS:$angularVersion",
```

Comment:
You need to add this value to the gradle.properties, if it's not the same as $pluginsVersion

Reply:
Added it now, but please note that locally I was only able to test with following setup: 

```gradle
pluginsVersion=233.9102.97
sassVersion=233.9102.195
angularVersion=233.9102.195
...
platformVersion=233.9102.97-EAP-SNAPSHOT
```